### PR TITLE
Update vendors.yaml to add ClickHouse

### PR DESCRIPTION
**Pull request by tbragin**
Hi! Adding ClickHouse to OpenTelemetry vendor list, per the vendor docs instructions.

Link to the documentation that details how your offering consumes OpenTelemetry natively via OTLP. 
https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/clickhouseexporter 
https://clickhouse.com/blog/storing-log-data-in-clickhouse-fluent-bit-vector-open-telemetry 
https://clickhouse.com/blog/storing-traces-and-spans-open-telemetry-in-clickhouse 
https://clickhouse.com/use-cases/logging-and-metrics

Link to your distribution, if applicable
N/A

Link that proves that your offering is open source, if applicable. An open source distribution does not qualify your offering to be marked “open source”. 
https://github.com/ClickHouse/ClickHouse/blob/master/LICENSE

GitHub handle or email address as a point of contact so that we can reach out in case we have questions 
@tbragin

### CODE REVIEW

File: data/ecosystem/vendors.yaml [hunk 1]
```diff
@@ -321,3 +321,10 @@
   contact: 'info@greptime.com'
   oss: true
   commercial: true
+name: ClickHouse
+  distribution: false
+  nativeOTLP: true
+  url: 'https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/clickhouseexporter'
+  contact: ''
```
Comment (svrnm) [inline data/ecosystem/vendors.yaml:328]:
```suggestion
  contact: 'https://github.com/tbragin'
```

File: data/ecosystem/vendors.yaml [hunk 2]
```diff
@@ -321,3 +321,10 @@
   contact: 'info@greptime.com'
   oss: true
   commercial: true
+name: ClickHouse
+  distribution: false
+  nativeOTLP: true
+  url: 'https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/clickhouseexporter'
```
Comment (svrnm) [inline data/ecosystem/vendors.yaml:327]: That doesn't fit, if this would be a support for native OTLP you would not need a vendor specific exporter

**Review by svrnm — CHANGES_REQUESTED** [review]

File: data/ecosystem/vendors.yaml (hunk 2 above)
Comment (tbragin) [reply data/ecosystem/vendors.yaml:327]: I must have misinterpreted what "native OTLP" means. If it means that we support OTLP protocol in ClickHouse database itself -- then yes, it is not the case, we do require an exporter. Then, we should change it to false.

Comment (svrnm) [reply data/ecosystem/vendors.yaml:327]: then set it to false please

Comment (tbragin) [reply data/ecosystem/vendors.yaml]: Yes, just wanted to triple check my understanding. I set "nativeOTLP: false".

**Review by svrnm — APPROVED** [review]

**Pull request merged**